src/include/txn_inline.h

```diff
@@ -224,6 +225,19 @@ __txn_next_op(WT_SESSION_IMPL *session, WT_TXN_OP **opp)
     op = &txn->mod[txn->mod_count++];
     WT_CLEAR(*op);
     op->btree = S2BT(session);
+
+    /*
+     * Store the ID of the latest transaction that is making an update. It can be used to determine
+     * if there is an active transaction on the btree. Retry if we race with another thread, but
+     * give up if that update is newer than this one.
```

Comment:
I'd prefer if this comment were rephrased from "but give up if that update is newer than this one", to "Only try to update the shared value if this is the newest transaction to do so" (or similar).
i.e: hopefully this shared variable update doesn't need to run all that often.

Reply:
Done.  Changed the third sentence to "Only try to update the shared value if this transaction is newer than the last transaction that updated it."